client/state/site-rename/actions.js

```diff
@@ -95,7 +95,7 @@ export const clearValidationError = siteId => dispatch => {
 	} );
 };
 
-export const requestSiteRename = ( siteId, newBlogName, discard ) => dispatch => {
+export const requestSiteRename = ( siteId, newBlogName, discard = true ) => dispatch => {
```

Comment:
I wouldn't do it in this PR, but I feel like this big thunk may benefit from porting related wpcom calls to the data-layer and experimenting with proposed flow changes by @jsnajdr p4TIVU-8SO-p2, if this form is localized to just one instance. Ping @Automattic/team-calypso if you're interested.

Reply:
Interested for sure! I'm about to be travelling and AFK-ing for most of the rest of the week so I'll get to that on Monday 👌 Thanks @gwwar!